app/config/galoy-instances.ts

```diff
@@ -0,0 +1,30 @@
+import { INetwork } from "@app/types/network"
+import { scriptHostname } from "@app/utils/helper"
+
+export type GaloyInstance = {
+  name: "BBW" | "Staging" | "Local"
+  network: INetwork
+  graphqlUri: string
+  graphqlWsUri: string
+}
+
+export const GALOY_INSTANCES: GaloyInstance[] = [
+  {
+    name: "BBW",
+    network: "mainnet",
```

Comment:
instead of hardcoding this, shouldn't this be fetch dynamically from the graphql endpoint?